cmd/pfcpctl/main.go

```diff
@@ -57,6 +58,10 @@ func main() {
 		os.Exit(0)
 	}
 
+	if *qfi > 255 {
```

Comment:
QFI is 6 bit long, so it shouldn't be greater than 64.

Reply:
8 if you count also the spare bits. I'm assuming that we don't care about spare bits?